### PR TITLE
Jh/cta-on-specific-lesson

### PR DESCRIPTION
**Pull request by joelhooks**
screenshot

CTA

### CODE REVIEW

File: src/pages/lessons/[slug].tsx [hunk 1]
```diff
@@ -52,6 +52,29 @@ import EmailCaptureCtaOverlay from '../../components/pages/lessons/overlay/email
 
 const tracer = getTracer('lesson-page')
 
+const specialLessons = {
```
Comment (joelhooks) [inline src/pages/lessons/[slug].tsx:55]: this is just a hack to identify a specific lesson to show some stuff when it's over

not very in depth, just noticed this lesson was getting a LOT of google flow

File: src/components/pages/lessons/overlay/watch-next-lesson-cta-overlay.tsx [hunk 2]
```diff
@@ -57,6 +61,39 @@ const WatchNextLessonCtaOverlay: React.FunctionComponent<{
...
+                      layout="fill"
+                    />
+                  </div>
+                  {content.title}
+                </a>
+              </Link>
+            )
+          })}
+        </div>
+      )}
```
Comment (joelhooks) [inline src/components/pages/lessons/overlay/watch-next-lesson-cta-overlay.tsx:96]: from a visual design perspective this is NOT GOOD

screenshot

**Comment by joelhooks** [comment]
/lessons/javascript-3-ways-to-update-the-content-of-an-array-of-objects-with-javascript

https://analytics.google.com/analytics/web/#/report/content-landing-pages/a36512724w249816875p231496770/_u.date00=20211029&_u.date01=20211029&explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.landingPagePath:~2Flessons~2Fjavascript-3-ways-to-update-the-content-of-an-array-of-objects-with-javascript

here's the GA

File: src/components/pages/lessons/overlay/watch-next-lesson-cta-overlay.tsx (hunk 2 above)
Comment (zacjones93) [reply src/components/pages/lessons/overlay/watch-next-lesson-cta-overlay.tsx:93]: I did a [quick redesign](https://www.figma.com/file/mzc7h7IRA9Vy9pU4cgm9kE/Lesson-Overlay-CTA?node-id=0%3A1). This separates the two different actions that you could take. 'watch again' / 'play next' being the usual CTA while choosing another course being the additional CTA.

Also, the contrast ratio on the overlay is failing AAA contrast with a score of 2.17 (< AA+). Updating the background to `#555A60` gives us a score of ~7.0. 


lesson overlay CTa

**Comment by joelhooks** [comment]
@zacjones93 I look forward to seeing that implemented!

**Pull request merged**